adot-testbed/app/src/test/resources/logs/log4j.log

```diff
@@ -0,0 +1 @@
+[otel.javaagent 2023-09-25 16:56:22:242 +0000] [OkHttp ConnectionPool] DEBUG okhttp3.internal.concurrent.TaskRunner - Q10002 run again after 300 s : OkHttp ConnectionPool
```

Comment:
I don't think this will work if you don't have a new line in the end of this line.